com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolume.Streaming.cs

```diff
@@ -50,20 +47,21 @@ void ComputeCellCameraDistance(Vector3 cameraPosition, DynamicArray<CellInfo> ce
 
         void ComputeStreamingScoreForBlending(DynamicArray<BlendingCellInfo> cells, bool areUploaded)
         {
+            float factor = scenarioBlendingFactor;
             for (int i = 0; i < cells.size; ++i)
             {
-                var cellInfo = cells[i].cellInfo;
-                if (m_ScenarioLerpFactor >= 1.0f && cells[i].blendingFactor >= 1.0f) // Finished blending
+                if (factor == cells[i].blendingFactor)
```

Comment:
technically i think it's fine since what we are testing here is if blendingFactor value didn't change since last frame, which should work with float equality
But epsilon range should work too

Reply:
then what about including the epsilong test in the public api setter ?